docs/stream.md

```diff
@@ -79,6 +79,23 @@ if __name__ == "__main__":
 stream_engine.remove_stream(stream)
 ```
 
+### Starting the stream with initial offsets
+If you want to start your consumption from certain offsets, you can include that in your stream instantiation.
```

Comment:
This needs more documentation, what happens if you provide 1 partition out of 3? Does it expect all partitions to be provided or you can initialize partially? Could you add an example scenario for this use case?

Reply:
I will document. The code will seek what it can, and ignore and log warning if it fails to find partition, topic, etc.